context/app/static/js/components/home-revision/AssayTypeBarChartContainer/hooks.js

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+import { useTheme } from '@material-ui/core/styles';
+
+import { getAssayTypeBarChartData } from './utils';
+
+function useChartPalette() {
+  const {
+    palette: { success, primary, secondary, error, warning, info },
+  } = useTheme();
+
+  const colorObjects = [success, primary, secondary, error, warning, info];
+
+  return [...colorObjects.map((c) => c.light), ...colorObjects.map((c) => c.main), ...colorObjects.map((c) => c.dark)];
```

Comment:
As I understand it, you need a palette for the bars, and since we've got a bunch of colors with different semantics for other parts of the site, they can be re-used here? I don't really care much myself... but my sense is that these uses are a little bit different: We don't actually want to draw attention to any particular color in the chart, but that is the point of some of these colors.

Reply:
@tsliaw, thoughts?

Reply:
As we discussed, we're not immediately concerned with everything pointed out by @mccalluc, but we could expect someone else to be concerned.